### ISSUE TITLE
CUB dataset train.txt and test.txt file issues

### ISSUE THREAD

**Issue by YanHanANU** [bnulihaixia/Deep_metric#11]
Hey, thank u very much for uploading your great work!
I got a small problem about reading your train.txt and test.txt files in Dataset folder. Firstly, it seems like u haven't uploaded any test.txt file. Secondly, for different dataset, the train and test txt files should be different, right? 
Did u forget to upload it or am I miss understanding ur work?
thank u very much again!
cheers

**Comment by bnulihaixia** [comment]
Sorry for late response, I was in travel recently. The train.txt and test.txt should be prepared by yourself.
It looks like this:




![screen shot 2018-09-25 at 9 26 44 am](https://user-images.githubusercontent.com/22806601/45987726-b2668480-c0a5-11e8-9547-db139e47d392.png)

the 1st column is the relative path of the image,  the 2nd is its label.

**Comment by YanHanANU** [comment]
thank u so much for ur reply.
I've fixed this file. However, the results for CUB dataset on my computer is not good. I was running the code with 'run_train_00.sh' file provided by u. But the accuracy is keeping decreasing from 50% to 0%. Do you have any idea of why this is happening? I am trying hard to fix this. Thank you so much for your work and suggestions!
cheers

**Comment by bnulihaixia** [comment]
Have you check your PyTorch version, this code only works for pytorch 0.2.03

**Comment by YanHanANU** [comment]
U said the performance is only 10% lower with pytorch 0.3.0? Cause I am using cuda9.1 and torch0.3.0, it is very difficult for me to downgrade pytroch to 0.2.0. But if that is the reason, i may have to reinstall cuda and pytorch.

Thank you so much for ur reply.

**Comment by zyyhl** [comment]
Hello, can you share the train.txt and test.txt files of CUB200? I think I have problems with the production, thank you.

**Comment by bnulihaixia** [comment]
I upload the cub and car datasets and train.txt and test.txt respectively  to Baidu Pan as below: 
链接:   https://pan.baidu.com/s/1LPHi72JPupkvUy_1OIn6yA 
提取码:    inmj 
You can get by yourself.
